packages/e2e/axe-selenium/src/main/java/org/talend/axeselenium/Reporter.java

```diff
@@ -0,0 +1,127 @@
+                    .append(i + 1)
+                    .append(") ")
+                    .append(violation.getString("help"));
+
+            if (violation.has("helpUrl")) {
+                sb
+                        .append(": ")
+                        .append(violation.getString("helpUrl"));
+            }
+
+            final JSONArray nodes = violation.getJSONArray("nodes");
+
+            for (int j = 0; j < nodes.length(); j++) {
+                final JSONObject node = nodes.getJSONObject(j);
+                sb
+                        .append(lineSeparator)
+                        .append("  ")
+                        .append(getOrdinal(j + 1))
+                        .append(") ")
+                        .append(node.getJSONArray("target"))
+                        .append(lineSeparator);
+
+                final JSONArray all = node.getJSONArray("all");
+                final JSONArray none = node.getJSONArray("none");
+
+                for (int k = 0; k < none.length(); k++) {
+                    all.put(none.getJSONObject(k));
+                }
+
+                appendFixes(sb, all, "Fix all of the following:");
+                appendFixes(sb, node.getJSONArray("any"), "Fix any of the following:");
+            }
+        }
+
+        return sb.toString();
+    }
+
+    private void appendFixes(final StringBuilder sb, final JSONArray arr, final String heading) {
+        if (arr != null && arr.length() > 0) {
+            sb
+                    .append("    ")
+                    .append(heading)
+                    .append(lineSeparator);
+
+            for (int i = 0; i < arr.length(); i++) {
+                JSONObject fix = arr.getJSONObject(i);
+
+                sb
+                        .append("      ")
+                        .append(fix.get("message"))
+                        .append(lineSeparator);
+            }
+
+            sb.append(lineSeparator);
+        }
+    }
+
+    /**
+     * Get an ordinal composed of letters, from a to z, then aa to az, the  ba to bz, ...
+     * @param number The ordinal index number
+     * @return The ordinal
+     */
+    private String getOrdinal(int number) {
+        String ordinal = "";
+
+        int mod;
+
+        while (number > 0) {
+            mod = (number - 1) % 26;
+            ordinal = (char) (mod + 97) + ordinal;
+            number = (number - mod) / 26;
+        }
+
+        return ordinal;
+    }
+
+    /**
+     * Writes a raw object out to a JSON file with the specified name.
+     *
+     * @param name   Desired filename, sans extension
+     * @param output Object to write. Most useful if you pass in either the Builder.analyze() response or the
+     *               violations array it contains.
+     */
+    public void writeResults(final String name, final Object output) {
+        try (PrintWriter out = new PrintWriter(name + ".json")) {
+            out.println(output.toString());
+        } catch (final IOException ignored) {}
```

Comment:
Handle the exception.

Reply:
👍  I let the method throw the exception to let the test fail.